packages/api/src/router/user.ts

```diff
@@ -81,7 +90,16 @@ export const userRouter = createTRPCRouter({
         .from(users)
         .where(eq(users.id, input.userId))
         .limit(1);
+      const existingUser = await ctx.db.query.users.findFirst({
+        where: eq(users.name, input.form.name.toLowerCase()),
```

Comment:
Is this case insensitive?

Reply:
not sure, kind of want to change the create user method to .toLowercase() usernames just in case